Lessen_6_hw/task_2.js

```diff
@@ -0,0 +1,90 @@
+/*
+1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+  и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+  Пиццы конкурента:
+  const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+*/
+
+function getEmployeeInfo(pizza) {
+	const competitorPizzas = [
+		'Peperoni',
+		'Caprichosa',
+		'Diablo',
+		'4 cheeses',
+		'hawai',
+	]
+	let unique = []
+	for (let i = 1; i < pizza.length; i++) {
+		if (competitorPizzas.indexOf(pizza[i]) === -1) {
+			unique.push(pizza[i])
+		}
+	}
+	if (unique.length > 0) {
+		return unique
+	} else {
+		return null
+	}
+}
+
+const myPizzas = [
+	'Peperoni',
+	'Caprichosa',
+	'Diablo',
+	'4 cheeses',
+	'hawai',
+	'Calzone',
+	'Carbonara',
+	'Marinara',
+	'Crudo',
+]
+
+const myPizzas_2 = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+console.log(getEmployeeInfo(myPizzas))
+
+// 2. Написать функцию, которая принимает предложение (слова разделенные только пробелами) в качестве параметра и выводит в консоль слово с наибольшим количеством букв.
+//   Если таких слов несколько - выводит их все.
+
+function moreLetters(string) {
+	string = string.split(' ')
+	let arr = []
+	let maxLengthWord = 0
+	for (let i = 0; i < string.length; i++) {
+		if (string[i].length > maxLengthWord) {
+			maxLengthWord = string[i].length
+		}
+	}
+	for (let i = 0; i < string.length; i++) {
+		if (string[i].length === maxLengthWord) {
+			arr.push(string[i])
+		}
+	}
```

Comment:
Use just one loop for this task
```suggestion
	for (let i = 0; i < string.length; i++) {
		if (string[i].length > maxLengthWord) {
			maxLengthWord = string[i].length
			arr = [string[i]]
		} else if (string[i].length === maxLengthWord) {
		    arr.push(string[i])
	     }
```